x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.test.tsx

```diff
@@ -746,6 +774,122 @@ describe('when on the endpoint list page', () => {
       expect(renderResult.getByTestId('endpointDetailsActionsButton')).not.toBeNull();
     });
 
+    describe('when showing Activity Log panel', () => {
+      let renderResult: ReturnType<typeof render>;
+      const agentId = 'some_agent_id';
+
+      let getMockData: () => ActivityLog;
+      beforeEach(async () => {
+        global.IntersectionObserver = jest.fn(() => ({
```

Comment:
can you use `window` instead of `global`

Reply:
done 6f34ef606c94b565e2a1bf8ef7a6077cb645744d